rqt_joint_trajectory_controller/src/rqt_joint_trajectory_controller/joint_limits_urdf.py

```diff
@@ -33,12 +33,15 @@ def get_joint_limits(key='robot_description', use_smallest_joint_limits=True):
             if jtype == 'fixed':
                 continue
             name = child.getAttribute('name')
+            try:
+                limit = child.getElementsByTagName('limit')[0]
+            except:
+                continue
```

Comment:
That makes sense. A warning would be nice for non-fixed joints that were skipped.